remix_exercises/Challenge0.sol

```diff
@@ -0,0 +1,14 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+interface Challenge0 {
+    function callMe() external;
+}
+
+contract Challenge0 {
+    event Winner(address);
+
+    function callMe() external {
```

Comment:
What do you think about adding in a `require(msg.value == 5 ether, "This function costs 5 ether!")`? This will teach them how to set the value in a transaction.

Reply:
Yeah that's a good idea.